apps/dev/app/api/auth/[...nextauth]/route.ts

```diff
@@ -0,0 +1,14 @@
+import NextAuth, { type NextAuthOptions } from "next-auth"
```

Comment:
Do you think we should start recommending using /auth instead of /api/auth? 🤔

Reply:
Maybe not for `next-auth`. Easier to migrate.

Reply:
When I export `authOptions` from this file nextjs complains that it doesn't match the spec for an app route. I am using `next@13.4.3` and `next-auth@4.22.1`. To fix it i just dont export the `authOptions`. I am wondering if that is an issue? What is the need to export options here, is it to just be able to import it somewhere else if you needed to utilize the config? Alternatively, [this article suggests](https://codevoweb.com/setup-and-use-nextauth-in-nextjs-13-app-directory/) to just define the config in a file outside of the app directory, then import it here in the route.

Reply:
The docs on [Nextjs Route Handlers](https://nextjs.org/docs/app/building-your-application/routing/router-handlers) seem to suggest that you can only export consts that match one of the HTTP METHODs from an App Route. So I believe this will need to be updated.